README.md

```diff
@@ -1,22 +1,59 @@
 # Elasticsearch
 
-This repository contains Ruby integrations for [Elasticsearch](https://www.elastic.co/products/elasticsearch):
+This repository contains Ruby integrations for [Elasticsearch](https://www.elastic.co/products/elasticsearch).
 
-* a client for connecting to an Elasticsearch cluster,
-* a Ruby API for the Elasticsearch's REST API,
-* a Ruby API for the X-Pack APIs,
-* various extensions and utilities.
+The [`elasticsearch`](https://github.com/elasticsearch/elasticsearch-ruby/tree/master/elasticsearch)
+library is a wrapper for two separate libraries:
+
+* [`elasticsearch-transport`](https://github.com/elasticsearch/elasticsearch-ruby/tree/master/elasticsearch-transport),
+  which provides a low-level Ruby client for connecting to an [Elasticsearch](https://www.elastic.co/products/elasticsearch) cluster
+* [`elasticsearch-api`](https://github.com/elasticsearch/elasticsearch-ruby/tree/master/elasticsearch-api),
+  which provides a Ruby API for the Elasticsearch RESTful API
+
+```ruby
+require 'elasticsearch'
+
+client = Elasticsearch::Client.new log: true
+
+# if you specify Elasticsearch host
+# client = Elasticsearch::Client.new url: 'http://localhost:9200', log: true
+
+client.transport.reload_connections!
+
+client.cluster.health
+
+client.search q: 'test'
+
+# etc.
+```
+
+Both of these libraries are extensively documented.
+**Please read the [`elasticsearch-transport`](http://rubydoc.info/gems/elasticsearch-transport) and the [`elasticsearch-api`](http://rubydoc.info/gems/elasticsearch-api) documentation carefully.**
+
+See also [`doc/examples`](https://github.com/elastic/elasticsearch-ruby/blob/master/docs/examples/README.md) for some practical examples.
+
+_Keep in mind, that for optimal performance, you should use a HTTP library which supports persistent
+("keep-alive") connections, e.g. [Patron](https://github.com/toland/patron) or [Typhoeus](https://github.com/typhoeus/typhoeus)._ These libraries are not dependencies of the elasticsearch gems, so be sure to define a dependency in your own application.
```

Comment:
Italics are bad for readability generally. I'd use bold to draw attention to the text. I also think you can remove `Keep in mind...`.

I think Elasticsearch would be capitalized here. Otherwise, it should be formatted as a literal. 
I'd also split the second sentence to make it a little more readable.
```suggestion
**For optimal performance, you should use a HTTP library which supports persistent
("keep-alive") connections, e.g. [Patron](https://github.com/toland/patron) or [Typhoeus](https://github.com/typhoeus/typhoeus).** These libraries are not dependencies of the Elasticsearch gems. Ensure you define a dependency for a HTTP library in your own application.
```